### ISSUE TITLE
Restrain search to a specific folder

### ISSUE THREAD

**Issue by tepelbaum** [imsnif/monocle#12]
Hi! Sorry if this is a silly question, but the README.md did not contain the information I was searching for: 

how do you restrain search to a specific folder?

Thanks for your work!! :)

**Comment by imsnif** [comment]
Sorry for the super late reply! The search folder picks up the CWD of the currently focused pane.

**Comment by tepelbaum** [comment]
No worry, thanks!  and (and I would like the same thing to be possible for the strider module), is it possible in a layout.kdl file to specify this cwd? 

Thanks again for your time!!! :)

**Comment by imsnif** [comment]
Yes - this will be possible for all plugins in v0.40.0 release hopefully in the coming two weeks or so.

**Comment by tepelbaum** [comment]
Awesome! Do not hesitate to close this issue once this new release comes out, that way I'll test it asap! :smile:
